.gitignore

```diff
@@ -1 +1,6 @@
 node_modules
+types/x-test-reporter.css.d.ts
+types/x-test-reporter.d.ts
+types/x-test-root.d.ts
+types/x-test-suite.d.ts
+types/x-test-tap.d.ts
```

Comment:
These aren’t meant to be imported directly, so there‘s no need to actually commit them to source yet. Only the `x-test.d.ts` file is really needed.